draft-meynell-panrg-scion-research-questions.md

```diff
@@ -196,9 +196,31 @@ This draft assumes the reader is familiar with some of the fundamental concepts
 
 ## ISD, AS identity
 
+The SCION protocol specifies 16 bits and 48 bits to identify the ISD and AS respectively.
+This identification is used, at the data-plane level, in every packet to fully address the sender and receiver,
+and at the control-plane level, to identify the PCB sender and hops.
+
+Whilst 48 bits for the AS will accommodate up to 2.81475e14 assignments which is
+likely to be more than sufficient for the future
+using 16 bits for the ISD only offers 65,536 possible assignments which is conceivably insufficient.
+Additionally, the justification of these figures is lacking at the moment.
+
+The following questions arise: (not comprehensive)
+
 * How many ASes do we expect in the SCION network model?
-* How many ISDs? What is the ontology of an ISD? Per geographic area only?
-* One AS belongs to many ISDs?
+* Can one AS belong to many ISDs?
+* Are AS numbers unique themselves? Or only unique in combination with an ISD?
+* How many ISDs do we expect?
+* What is the ontology of an ISD?
+  * Per geographic area?
+  * Per legal jurisdiction?
+  * Per capacity tier?
+  * Per "type"? (meaning: any grouping that makes sense to their members)
+  * Possible combinations of any previous "types"?
+* Note that ISDs require CORE links to other ISDs.
```

Comment:
We don't really mandate connectivity in the specification. Is there a reason why you say "require"?

Reply:
Core AS from ISD1 MUST send PCBs to core AS from ISD2.
That's where the connectivity requirement comes from.
But I don't know if I should change the wording, so that it is clear that both ISDs must be able to communicate via their core ASes, in principle without intermediaries.